backend/Makefile

```diff
@@ -53,6 +53,12 @@ db/connect:
 	# ARGS usage: AWS_PROFILE=single-cell-dev DEPLOYMENT_STAGE=dev make db/connect ARGS="-c \"select * from dataset_artifact where filetype='CXG'\""
 	PGPASSWORD=${DB_PW} psql --dbname corpora_${DEPLOYMENT_STAGE} --username corpora_${DEPLOYMENT_STAGE} --host 0.0.0.0 $(ARGS)
 
+db/rdev-connect:
```

Comment:
Not needed by this PR, but this is a useful command if you want to connect to the rdev database.

Reply:
does `db/tunnel` with with this or do we need an `db/rdev-tunnel` command too?

Reply:
Not sure because I keep my tunnels in a separate folder. I'll give it a try. Also this requires `DB_PW` to be provided externally since the way the secret is encoded is different from the other environments. I will try to improve this a bit.

Reply:
I'd say it's a "nice to have".  But if we don't have a team-wide mechanism for tunneling to rdev, we should probably just remove this for now.

Reply:
I removed it for now as the tunnel doesn't work either. Will repackage this as a script that can be shared among devs.